index.html

```diff
@@ -26,30 +23,30 @@
       <nav class="desktop-navbar">
         <ul class="desktop-navbar-links">
           <li class="actual-links">
-            <a href=""> Portfolio </a>
+            <a class="nav-links" href="#portfolio"> Portfolio </a>
           </li>
           <li class="actual-links">
-            <a href=""> About </a>
+            <a class="nav-links" href="#about"> About </a>
           </li>
           <li class="actual-links">
-            <a href=""> Contact </a>
+            <a class="nav-links" href="#contact"> Contact </a>
           </li>
           <li>
-            <a href="">
+            <a href="#contact">
               <img
                 class="nav-email"
-                src="./icons/email-icon.svg"
+                src="./images/email-icon.svg"
                 alt="email icon"
               />
             </a>
           </li>
         </ul>
       </nav>
```

Comment:
- [ ] Well done, when I click on a menu option, it leads me to the part mentioned (example: **About/Portfolio/contact**) :100: But it's impossible to scroll up or down afterward. I have to refresh the page, please fix this to give the user a pleasant user experience. :smile: 

https://www.loom.com/share/cc06d0baaae34627b6a9cd7d219be9b1